grafana/src/main/java/org/apache/iotdb/web/grafana/controller/DatabaseConnectController.java

```diff
@@ -69,15 +68,12 @@ public void testDataConnection(HttpServletResponse response) throws IOException
   /**
    * get metrics numbers in JSON string structure.
    *
-   * @param request http request
-   * @param response http response
    * @return metrics numbers in JSON string structure
    */
   @RequestMapping(value = "/search")
   @ResponseBody
-  public String metricFindQuery(HttpServletRequest request, HttpServletResponse response) {
+  public String metricFindQuery() {
```

Comment:
Why can we make this change here?

Reply:
In method body, we have not used the `request` param and the response's default code is 200, so we do not need these two params.